flexible_event_config.md

```diff
@@ -212,15 +217,46 @@ Triggers could be registered with the value field set, which are summed up and b
 { "event_trigger_data": [{"trigger_data": "0", "value": 4}] }
 ```
 
-The values are summed (to 8) and reported as bucket [5, 9]
+The values are summed (to 8) and reported in the following reports after 7 days + 1 hour:
```

Comment:
I'm still having a bit of difficulty understanding the mapping between triggers and reports. To help clarify, let's say each of the above triggers has a different `debug_key`. What would the reports look like?

Reply:
Hm I think we will need to spend some extra thought specifying how `debug_key` will compose with flexible event. This is because we lose the 1:1 mapping from triggers --> reports. With this proposal, multiple triggers can contribute to a single report, as they "aggregate" together within a particular summary bucket.

We could do a few things:
- keep track of all triggers that contributed to a report and emit all of their debug keys in a list.
- Emit the debug key of the last trigger which 
- etc

Reply:
I think that a list of all them is the most natural.

Separate from the keys themselves, though, we will have to consider the format, semantics, and benefit of [attribution-success debug reports](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#attribution-success-debugging-reports).